docs-samples/data-science/semantic-link-samples/great_expectations_tutorial.ipynb

```diff
@@ -0,0 +1,2642 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "id": "804ef10b-9098-40c8-8bcc-db71858015ad",
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.livy.statement-meta+json": {
+              "execution_finish_time": "2023-10-06T16:06:03.817579Z",
+              "execution_start_time": "2023-10-06T16:06:03.3836242Z",
+              "livy_statement_state": "available",
+              "parent_msg_id": "2e8a6e83-6c20-4791-9d8b-d0d92d64e47b",
+              "queued_time": "2023-10-06T16:06:02.998663Z",
+              "session_id": "efb436d1-09b2-43ab-a318-5b84c484989f",
+              "session_start_time": null,
+              "spark_jobs": {
+                "jobs": [],
+                "limit": 20,
+                "numbers": {
+                  "FAILED": 0,
+                  "RUNNING": 0,
+                  "SUCCEEDED": 0,
+                  "UNKNOWN": 0
+                },
+                "rule": "ALL_DESC"
+              },
+              "spark_pool": null,
+              "state": "finished",
+              "statement_id": 23
+            },
+            "text/plain": [
+              "StatementMeta(, efb436d1-09b2-43ab-a318-5b84c484989f, 23, Finished, Available)"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Created temporary directory '/tmp/tmpl3_woi23' for ephemeral docs site\n"
+          ]
+        }
+      ],
+      "source": [
+        "ds.add_powerbi_table_asset(\"Store Asset\", table=\"Store\")\n",
+        "\n",
+        "ds.add_powerbi_measure_asset(\n",
+        "    \"Total Units Asset\",\n",
+        "    measure=\"TotalUnits\",\n",
+        "    groupby_columns=[\"Time[FiscalYear]\", \"Time[FiscalMonth]\"]\n",
+        ")\n",
+        "\n",
+        "ds.add_powerbi_dax_asset(\n",
+        "    \"Total Units YoY Asset\",\n",
+        "    dax_string=\"\"\"\n",
+        "EVALUATE SUMMARIZECOLUMNS(\n",
+        "    'Time'[FiscalYear],\n",
+        "    'Time'[FiscalMonth],\n",
+        "    \"Total Units Ratio\", DIVIDE([Total Units This Year], [Total Units Last Year])\n",
+        ")    \n",
+        "\"\"\")\n",
+        "\n",
+        "# https://dax.tips/2019/11/28/clean-data-faster-reports/\n",
+        "ds.add_powerbi_dax_asset(\n",
+        "    \"Referential Integrity Violation\",\n",
+        "    dax_string=\"\"\"\n",
+        "SELECT\n",
+        "    [Database_name],\n",
+        "    [Dimension_Name],\n",
+        "    [RIVIOLATION_COUNT]\n",
+        "FROM $SYSTEM.DISCOVER_STORAGE_TABLES\n",
```

Comment:
how in familiar with DAX/DMV are we expecting people who use this tutorial to be? should we add an explanation of what these queries are above this cell?

Reply:
we can assume familiarity, but would still add some links in the decscription.